packages/website/src/components/ErrorsViewer.tsx

```diff
@@ -91,11 +91,21 @@ export default function ErrorsViewer({
 
   return (
     <div className={styles.list}>
-      {model?.map(([group, data]) => {
+      {value?.map(({ group, uri, items }) => {
         return (
           <div className="margin-top--sm" key={group}>
-            <h4>{group}</h4>
-            {data.map((item, index) => (
+            <h4>
+              {group}
+              {uri && (
+                <>
+                  {' - '}
+                  <a href={uri} target="_blank">
+                    details <LinkIcon width={13.5} height={13.5} />
```

Comment:
Nit: I personally associate "details" with per-instance details. Maybe that's just me.

```suggestion
                    docs <LinkIcon width={13.5} height={13.5} />
```